### PR TITLE
Em magento bulkapi

### PR DESCRIPTION
**Pull request by erikmarr**
## This PR is a:

- [ x] New topic
- [ ] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will demonstrate how to use the REST bulk API.

This PR merges in @nuzil new topics about using the REST bulk API.

List all affected URLs 

- https://devdocs.magento.com/guides/v2.3/rest/tutorials/index.html

whatsnew
Added information about how to use the REST bulk API.

### CODE REVIEW

File: guides/v2.3/rest/tutorials/index.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ The REST tutorials provide an introduction to Magento web APIs. In general, the
 
 * The [**configurable product** tutorial]({{ page.baseurl }}/rest/tutorials/configurable-product/config-product-intro.html) helps you plan then create a configurable product and its component simple products.
 
+* The [**bulk API configurable product** tutorial]({{ page.baseurl }}/rest/tutorials/bulk-configurable-product/config-product-intro.html) demonstrates how to perform actions on a large scale; such as, creating multiple customers and products, changing prices across an inventory, and assigning large groups of products to a specific warehouse, all within a single call.
```
Comment (keharper) [inline guides/v2.3/rest/tutorials/index.md]: This tutorial only shows how to perform one task: creating configurable products. Rewrite this paragraph.

Comment (erikmarr) [reply guides/v2.3/rest/tutorials/index.md]: rewrote to be more specific about what the tutorial is actually demonstrating.

**Comment by erikmarr** [comment]
Related issues and PRs:
Issue: #3508
PRs: #3612, #3580

**Comment by erikmarr** [comment]
running tests

**Comment by erikmarr** [comment]
Thanks @nuzil for your contribution!

**Comment by dshevtsov** [comment]
@erikmarr :
- incorporate references to new topics in whatsnew, so that a user could easily find the topics
- remove branches after merging them

**Pull request merged**